.gitignore

```diff
@@ -36,4 +36,13 @@ site
 *.local.yaml
 
 # Sensitive credentials
-*-credentials.yaml
\ No newline at end of file
+*-credentials.yaml
+
+# Yarn files
+.pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

Comment:
Where did this selection of ignores/non-ignores come from?  Is that an official list from somewhere?

Reply:
This info comes from an internal backstage tutorial: https://backstage.io/docs/tutorials/yarn-migration

This was already used when migrating to yarn-3, and now to yarn-4 as well. In this repo I just skipped some steps which were not really needed for our setup.

Reply:
Good to know.  https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored has more info about these files and which to ignore, I found that helpful.